client/src/app/TabsProvider.js

```diff
@@ -344,7 +344,13 @@ export default class TabsProvider {
         exports: {},
         extensions: [ 'form' ],
         canOpen(file) {
-          return file.name.endsWith('.form');
+          const {
+            contents
+          } = file;
+
+          const executionPlatform = JSON.parse(contents).executionPlatform;
```

Comment:
Let's make sure this does not blow up when `contents` cannot be parsed. `JSON.parse` will throw an error in that case. You can try out to pass a BPMN file here.

Reply:
Perhaps a broken JSON?

`{ myBrokenJson: true }`